src/Core/src/Graphics/PaintExtensions.Tizen.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Microsoft.Maui.Graphics
+{
+	public static partial class PaintExtensions
+	{
+		public static IDrawable ToDrawable(this Paint paint)
+		{
+			return new BackgroundDrawable(paint);
+		}
+
+		class BackgroundDrawable : IDrawable
+		{
+			Paint _paint;
+			public BackgroundDrawable(Paint paint)
+			{
+				// Workaround.
+				// I think, SolidPaint violate LSP
+				// When Paint was used on Canvas.SetFillPaint, BackgroundColor was referred
+				// But SolidPaint do not update BackgroundColor when Color was updated even though it has same meaning.
```

Comment:
I see this comment. Is it a bug in Maui.Graphcs or is some other place? Maybe it is a bug in some extension method somewhere in Controls?

Reply:
Yes. It is related to https://github.com/dotnet/Microsoft.Maui.Graphics/commit/de0493874c4fe0f74d26ca3a8564e79bfb629297 and we submitted the PR. (yes, it already merged. but not published yet.)

Reply:
Ah, it might be on the dotnet6 feed: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json

I think we might be able to get that in with this PR: https://github.com/dotnet/maui/pull/2346

Reply:
done. Thanks!